proc_scripts/plotting.py

```diff
@@ -68,12 +68,24 @@ class fl_mod(figlist_var):
     plotname:   string with name of plot 
     s:          nddata being analyzed
     thisrange:  range along x axis to be analyzed
+    f:          fitdata 
+                data (a fitting instance), on which `f.fit()` has already been run
+    guess:      None or nddata
+                The result of `s.settoguess();s.eval(100)` where 100 can be
+                any integer.
+                Used to display the guess on the plot as well.
+    name:       str
+                the name of the plot
 
     Returns
     =======
     plot image side by side with the cropped log 
 
     """
+    def next(self, *arg, **kwargs):
+        kwargs.update({"figsize": (9, 6), "legend": True})
+        super().next(*arg, **kwargs)
```

Comment:
Just add `return` in front of `super` so that it returns the same thing that the parent class does

Reply:
okay sorry I had deleted on accident but I put back in. I am guessing this defines the plot sizing? Do I need a doc string for this as well?